TrackletAlgorithm/MatchCalculator.h

```diff
@@ -314,30 +319,30 @@ void MatchCalculator(BXType bx,
   static const uint8_t shift_L1L2 = 0;
```

Comment:
these are still static const -- can't they be constexpr?

Reply:
Moved to MatchCalculator_parameters.h